src/python/pants/core/goals/repl.py

```diff
@@ -38,7 +38,7 @@ def in_chroot(self, relpath: str) -> str:
 
 
 class ReplSubsystem(GoalSubsystem):
-    """Opens a REPL."""
+    """Open a REPL with the specified code loaded."""
```

Comment:
I don't think this is necessarily obvious. A user DMed me today asking if it's possible to load code in a REPL.

Reply:
The code isn't loaded, exactly. I's on the PYTHONPATH, i.e., it's loadABLE.